### PR TITLE
add secret message support

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/help.py [hunk 1]
```diff
@@ -60,7 +61,6 @@ async def helpme(message: Message) -> None:  # pylint: disable=missing-function-
                      or plugins[key].enabled_commands[0].name.lstrip(Config.CMD_TRIGGER) != key)):
             commands = plugins[key].enabled_commands
             out_str = f"""⚔ <b><u>(<code>{len(commands)}</code>) Command(s) Available</u></b>
-
```
Comment (rking32) [inline userge/plugins/help.py:63]: why you removed these spaces. they are required to a clear view.

File: userge/plugins/help.py [hunk 2]
```diff
@@ -268,7 +280,6 @@ def plugin_data(cur_pos: str, p_num: int = 0):
         pos_list = cur_pos.split('|')
         plg = userge.manager.plugins[pos_list[2]]
         text = f"""🗃 **--Plugin Status--** 🗃
-
```
Comment (rking32) [inline userge/plugins/help.py:271]: here also

File: userge/plugins/help.py (hunk 1 above)
Comment (Krishna-Singhal) [reply userge/plugins/help.py:63]: when I make changes, it's automatically removed 🤷, wait let me add 😁

**Comment by rking32** [comment]
now code nice.
but we should add multi msg support

**Comment by Krishna-Singhal** [comment]
> now code nice.
> but we should add multi msg support

Will be a little mess.

**Comment by Krishna-Singhal** [comment]
https://t.me/usergeot/514338

**Pull request merged**